### ISSUE TITLE
[Feature]: Implement an AI Pipleline for CIFAR10-Dataset

### ISSUE THREAD

**Issue by Mikatech** [PoCInnovation/SpaceVector#5]
### Description

You have the [CIFAR-10](https://www.cs.toronto.edu/~kriz/cifar.html) dataset which contains 60,000 images, and 10 different labels.

Implement an application that uses computer vision. 🏅 

### Expected behaviour

Your application should:

- [x] getting started with python3 
- [x] take as parameter an argument which is a folder
- [x] perform start-up training
- [x] display the evolution of the loss and accuracy during training
- [x] Make predictions once trained on all the images that the folder passed in parameter contains.
- [x] Write a predictions ``output`` file that maps each image name to its label
- [x] Make an application Gradio

Bonus ideas

- Propose to the user to choose his hyperparameters 
- implement a graphical interface using [Streamlit](https://streamlit.io/) or [Gradio](https://gradio.app/)
- Integrate an AI stack like: [Hydra](https://hydra.cc/), [Optuna](https://optuna.org/) & [DVC](https://dvc.org/)

### Environment

/

### Proposed solution

/
